src/main/java/org/bytedeco/javacv/FFmpegFrameGrabber.java

```diff
@@ -1551,4 +1565,14 @@ public synchronized AVPacket grabPacket() throws Exception {
 
         return pkt;
     }
+
+
+    public void setVideoDisposition(int value){
+        disposition = value;
+    }
+
+    public int getVideoDisposition(){
+        return disposition;
+    }
```

Comment:
Let's move this property inside FrameGrabber, along the rest like videoCodec, etc.